### ISSUE TITLE
Missing Error Report

### ISSUE THREAD

**Issue by domainexpert** [tracer-x/klee#219]
For  [this example](https://github.com/tracer-x/klee-examples/blob/master/scalability/veritesting_plus.c), Tracer-X does  not report an error while LLBMC reports an assertion failure. Tracer-X finished in 50s with the following stats (ran with `-exit-on-error` option):
```
KLEE: done: Subsumption statistics
KLEE: done:     Time for actual solver calls in subsumption check (ms) = 0.259
KLEE: done:     Number of solver calls for subsumption check (failed) = 1 (0)
KLEE: done:     Concrete store expression build time (ms) = 18831
KLEE: done:     Symbolic store expression build time (ms) = 0.646
KLEE: done:     Solver access time (ms) = 1387.66
KLEE: done:     Average table entries per subsumption checkpoint = 4058.33
KLEE: done:     Number of subsumption checks = 19975
KLEE: done:     Average solver calls per subsumption check = 0.00
KLEE: done: TxTree method execution times (ms):
KLEE: done:     setCurrentINode = 18.916
KLEE: done:     remove = 190.975
KLEE: done:     subsumptionCheck = 24080.1
KLEE: done:     markPathCondition = 34.217
KLEE: done:     split = 102.459
KLEE: done:     executeOnNode = 150.583
KLEE: done:     executeMemoryOperation = 707.027
KLEE: done: TxTreeNode method execution times (ms):
KLEE: done:     getInterpolant = 15.842
KLEE: done:     addConstraintTime = 10.213
KLEE: done:     splitTime = 100.713
KLEE: done:     execute = 141.524
KLEE: done:     bindCallArguments = 0
KLEE: done:     bindReturnValue = 0
KLEE: done:     getStoredExpressions = 843.232
KLEE: done:     getStoredCoreExpressions = 55.736

KLEE: done: total instructions = 223057
KLEE: done: completed paths = 9988, among which
KLEE: done:     early-terminating paths (instruction time limit, solver timeout, max-depth reached) = 0
KLEE: done:     average branching depth of completed paths = 100
KLEE: done:     average branching depth of subsumed paths = 94.6954
KLEE: done:     average instructions of completed paths = 1824
KLEE: done:     average instructions of subsumed paths = 1712.02
KLEE: done:     subsumed paths = 7800
KLEE: done:     error paths = 0
KLEE: done:     program exit paths = 2188
KLEE: done: generated tests = 9988, among which
KLEE: done:     early-terminating tests (instruction time limit, solver timeout, max-depth reached) = 0
KLEE: done:     subsumed tests = 7800
KLEE: done:     error tests = 0
KLEE: done:     program exit tests = 2188
```
LLBMC finished in 31.4s reporting the assertion violation.
```
This is the Low-Level Bounded Model Checker LLBMC (version 2013.1).
Running LLBMC with function call depth 0 and at most 0 iterations for loops with non-derivable trip count.

Performing bounded model checking ...
  Performing transformations on LLVM IR ...            0.000 sec [DONE]
   
  Performing logical encoding ...                      0.020 sec [DONE]
   
  Performing formula transformations ...               0.024 sec [DONE]
   
  Running SMT solver (STP with MiniSat) ...           31.352 sec [DONE]
   
                                                      31.396 sec [DONE]
 
Result:
=======

Error detected.

Error synopsis:
===============

Assertion failed: Custom assertion (assert or __llbmc_assert) does not hold.

Error location:
===============

Error occurs in basic block "return" of function "main".

Source file: /home/dcsandr/software/klee-examples/scalability/veritesting_plus.c
Line number: 38
Error context:

  33:         counter += i;
  34:       }
  35:   }
  36:
  37: #ifdef LLBMC
  38:   __llbmc_assert(counter != start + 501);
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  39: #else
  40:   klee_assert(counter != start + 501);
  41: #endif
  42:  
  43:   return 0;

Stack trace:
============

#0 i32 @main(i32 %argc=1, i8** %argv)
```
Normal KLEE run has not yet been done due to exponential blowup of paths.

**Comment by domainexpert** [comment]
LLBMC options used was `--ignore-missing-function-bodies -function-name=main -no-overflow-checks -max-loop-iterations=0 -max-function-call-depth=0 -counterexample`. The last option is for including the counterexample in the output file.

**Comment by domainexpert** [comment]
[This](https://github.com/tracer-x/klee/files/1291180/counter.txt) shows the change in the values of `counter` when `start` was initialized to 0 (even when `start` is initialized to 0 the problem remains). The change in `counter` value is perhaps a clearer evidence of over-subsuming, and can be used to investigate where is it that Tracer-X KLEE over subsumes.

**Comment by domainexpert** [comment]
Resolved via #303.
